### DISCUSSION TITLE
What does red square or red tilde before line number mean?

### DISCUSSION THREAD

**Discussion by stepanzak** [NvChad/NvChad#1857] [Q&A] [answered]
I can't find any info on what does this red square on line 12 mean:
![Screenshot_20230325_202643](https://user-images.githubusercontent.com/99094176/227737444-a1c16da1-004f-447e-86e7-bbcad81d437d.png)

~~Also, with this chadrc.lua, nvdash is not showing up on startup. Is that because of that square?~~
When I comment out the whole line, the square changes to red ~.
Thanks for any help.

**Comment by siduck** [comment, marked as answer]
hi @stepanzak the red tilde comes from the gitsigns.nvim plugin it indicates that the line on it was deleted

**Reply by stepanzak** [reply]
Thanks!
The red square is from that plugin too?

**Reply by siduck** [reply]
@stepanzak i dont think so, it had a different icon before. If you check our plugin/configs/others.lua, you can find the icon used in gitsigns options table there
